src/main/java/com/pscheduler/server/repository/GPACourseRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.pscheduler.server.repository;
+
+import com.pscheduler.server.model.GPACourse;
+import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+import org.springframework.data.repository.CrudRepository;
+
+
+@RepositoryRestResource(collectionResourceRel="gpa")
```

Comment:
Just tested your branch `collectionResourceRel` is needed, but also add `path="gpa"`. if you check `http://localhost:8080/api/` with your change notice how the link is still `gPAcourses`

Reply:
Fixed